api/v1alpha1/flowcollector_types.go

```diff
@@ -215,6 +215,14 @@ type FlowCollectorKafka struct {
 	// tls client configuration.
 	// +optional
 	TLS ClientTLS `json:"tls"`
+
+	//+kubebuilder:default:=100
+	// pullQueueCapacity defines the capacity of the internal message queue used in the Kafka consumer client.
+	PullQueueCapacity int `json:"pullQueueCapacity"`
```

Comment:
maybe we should rather name it `ConsumerQueueCapacity` ? (that would be more consistent with the other CRD update PR)

Reply:
I think we need to specify QueueCapacity, BatchSize and LingerMS as a subsection of each component: for the agent and for FLP.

BTW I think LingerMS in only for Producers (the agent, in this case)